.travis.yml

```diff
@@ -29,8 +29,6 @@ script:
 
 cache:
```

Comment:
Don't we need to remove this line and the next one as well? See #3972

Reply:
We want to cache storing `node_modules`, which is what
```cache: 
  - npm: true
```
does.